app/renderer/ui/components/main/sections/wallet/tabs/transactions/index.tsx

```diff
@@ -28,6 +27,13 @@ const styles = require("./style.scss")
 class Transactions extends TabComponent<any> {
   // tslint:disable-next-line:prefer-function-over-method completed-docs
   public render() {
+    const confirmedOptions = ["Option 1", "Option 2", "Option 3"].map((opt: string) => (
+      {
+        text: opt,
+        onClick: () => this.props.services.showUnimplementedMessage()
```

Comment:
Also here, sorry (just pointing out the places where it fails so we can check afterwards that they are all fixed and not miss any of them :+1:)